samples/training/train-on-amlcompute/train.R

```diff
@@ -37,5 +37,10 @@ message(conf_matrix)
 
 log_metric_to_run(metric, conf_matrix$overall["Accuracy"])
 
+output_dir = "outputs"
+if (!dir.exists(output_dir)){
+  dir.create(output_dir)
+}
```

Comment:
"outputs" should pre-exist for every run. we probably don't the need the check here.

Reply:
what if user runs the script locally first?

Reply:
for local run yes. this sample is for amlcompute though.